PERLUR/app.exchange.perlur.cloud
# Fixing issues with vuejs render

**Context**

The Vue.js wasn't working as it was not part of the HTML.

**Summary of Changes**

This is a just quick fix to make the project work. The is not the ideal solution, but good enough to start with.

**Testing Instructions**

Run the app and you should see the login page with Vue.js validating the login form. In detail, you can use Chrome's console to see the console output and all warnings.
